src/Cody.VisualStudio/CodyPackage.cs

```diff
@@ -154,7 +154,8 @@ private void ReportSentryVsVersion()
         {
             SentrySdk.ConfigureScope(scope =>
             {
-                scope.SetTag("vs", VsVersionService.DisplayVersion);
+                scope.SetTag("vs_version", VsVersionService.DisplayVersion);
```

Comment:
Let's leave `vs` tag, it's quite explicit and other default Sentry tags doesn't use `version` postfix - like e. g. `os`, `release`, `runtime` etc.

Also for agent. let's use `agent` without `version` postfix for consistency.